source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -175,13 +203,16 @@ Supported Operating Systems and Architectures:
 - Only install scripts if they are used by a specific platform
 - Make shortcuts use the app icon
 - Add AppArmor file for electron apps for Ubuntu 24.04
-- Fix icon in dock on Ubuntu 24.04
 
 ## Visual Studio Code Extension
 
 - Add :doc:`Dependency Manager extension </docs/software/vscode-overview/3rd-party-libraries>` for easier finding and updating of 3rd party libraries
```

Comment:
```suggestion
- Add :doc:`Dependency Manager extension </docs/software/vscode-overview/3rd-party-libraries>` for easier finding and updating of 3rd party libraries
- The Dependency Manager will now automatically initiate a build when the utility loses focus, but only if the user has made changes.
```

Reply:
Normally we only put things in the changelog that changed from year to year, since that's what's relevant come January.

Reply:
I think it depends on how you look at it.  I am thinking about this as a change from what people are expecting coming from past years which is a pop up that prompts them to build.  We didn't include anything about this previously because the functionality prior to this matched that expectation from previous years.

The new functionality is something completely different that doesn't match the expectations of people coming from last year.  It now builds automatically with no prompt.